docs/docs/smart-contracts/PSP37/Extensions/mintable.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Next time use the "rename" feature of the github=) Instead of creating a new folder and copying there files. Because you lose the history of commits.

If you can do it for that PR, it will be excellent=)